tests/api/cluster/test_user.py

```diff
@@ -0,0 +1,48 @@
+from unittest.mock import call
+
+import pytest
+
+from controlpanel.api import cluster
+
+
+def test_iam_role_name(users):
+    assert cluster.User(users['normal_user']).iam_role_name == 'test_user_bob'
+
+
+def test_create(aws, helm, settings, users):
+    user = users['normal_user']
+    cluster.User(user).create()
+
+    aws.create_user_role.assert_called_with(user)
+    expected_calls = [
+        call(
+            f'init-user-{user.slug}',
+            'mojanalytics/init-user',
+            f"--set=NFSHostname={settings.NFS_HOSTNAME},"
+            f"Username={user.slug},"
+            f"Email={user.email},"
+            f"Fullname={user.get_full_name()},"
```

Comment:
I see that `models.User.get_full_name()` returns `self.name`, this is few months old, I wonder what was the rationale for this method and if needed/nicer to have if it could be a property (as `joe.full_name` reads better than `joe.get_full_name()` IMHO).

But as this is not new, no need to change, it's just an observation.